pom.xml

```diff
@@ -34,10 +34,10 @@
   </scm>
   <properties>
     <compiler-plugin.version>3.8.1</compiler-plugin.version>
-    <maven.compiler.release>11</maven.compiler.release>
+    <maven.compiler.release>17</maven.compiler.release>
```

Comment:
Is it normal to sneak in a Java 17 requirement?

Reply:
@FroMage it didn't work with 11 I suppose Quarkus 3.7 is dropping j11 support

Reply:
> As previously announced, the minimum Java version for Quarkus 3.7 will be Java 17.

Haha, OK cool :)